src/components/AnnotationViewer.vue

```diff
@@ -292,6 +311,82 @@ export default class AnnotationViewer extends Vue {
       this.drawNewConnections(displayedIds);
     }
     this.annotationLayer.draw();
+    this.drawTooltips();
+  }
+
+  drawTooltips() {
+    const oldFeatures = this.textLayer.features();
+
+    if (this.shouldDrawTooltips) {
+      // Avoid checking the store for each annotation
+      const all = this.tooltipOnAll;
+      const selected = this.tooltipOnSelected;
+      const hovered = this.tooltipOnHovered;
+
+      const displayedAnnotations = this.annotationLayer
+        .annotations()
+        .map((a: any) => a.options("storedAnnotation"))
+        .filter((a: any) => {
+          return (
+            a &&
+            (all ||
+              (hovered && this.hoveredAnnotationId === a.id) ||
+              (selected && this.isAnnotationSelected(a.id)))
+          );
+        });
+
+      // One text feature per line as in https://opengeoscience.github.io/geojs/tutorials/text/
+      // Centroid is computed once per new line (optimization needed?)
+      // TODO: More performance with renderThreshold https://opengeoscience.github.io/geojs/apidocs/geo.textFeature.html#.styleSpec
+      const baseStyle = {
+        fontSize: "12px",
+        fontFamily: "sans-serif",
+        textAlign: "center",
+        textBaseline: "middle",
+        color: "black",
+        textStrokeColor: "#FFF8",
+        textStrokeWidth: 2
```

Comment:
Is the text shown on top of a white/transparent rectangle? 
Because, in my opinion, it is possible that we won't be able to read information as the text font color is black, and we are working with black/white images..

Reply:
It is black with a half-transparent white stroke for readability.
I didn't find any option to add a background color for text.